webui/apps/docs/src/content/docs/docs/howto/authenticate_llm.md

```diff
@@ -49,4 +49,4 @@ MISTRAL_API_KEY=yourmistralapikey
 
 Restart Eidolon. Agents can now authenticate with your preferred LLMs.
 
->Note: if the `.env` file does not exist in your application root directory, create it as a plain text file. 
+>Note: if the `.env` file does not exist in your application root directory, create it as a plain text file. Always restart after editing the `.env` file.
```

Comment:
We should add this to our file watcher so that the server automatically restarts when in watch mode.

I'll file an issue for that, and then we can revert this change when that goes in.